vllm/platforms/cpu.py

```diff
@@ -62,12 +62,12 @@ def check_and_update_config(cls, vllm_config: VllmConfig) -> None:
 
         if kv_cache_space >= 0:
             if kv_cache_space == 0:
-                cache_config.cpu_kvcache_space_bytes = 4 * GiB_bytes  # type: ignore
```

Comment:
This particular field is being used in cpu worker.

Reply:
Ok, I'll revert this then, thanks!